CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.18.0] - 2020-11-24
+
 ### Added
```

Comment:
Since we have only added it should be 0.17.1 according to semver.

Reply:
ok :)